<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPackageDialog</class>
 <widget class="QDialog" name="AddPackageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>573</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>665</width>
    <height>575</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="installWarningLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>安装注意事项</string>
   </property>
  </widget>
  <widget class="QLabel" name="installResultLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>安装结果</string>
   </property>
  </widget>
  <widget class="QLabel" name="productNameLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>产品完整中文名称(含版本号)：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="installResultLineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>480</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="installReasonLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>安装原因</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="installReasonLineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>460</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="installWarningLineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>500</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="productNameLineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>661</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>软件开发测试时软件包版本信息及使用的专用信息设备版本信息</string>
   </property>
   <widget class="QLabel" name="developmentOsVersionLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>系统版本：</string>
    </property>
   </widget>
   <widget class="QLabel" name="developmentIoModuleVersionLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IO保密管理模块版本：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="developmentFirmwareVersionLineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="developmentSocVersionLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>安全卡版本：</string>
    </property>
   </widget>
   <widget class="QLabel" name="pkgArchlabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>软件包架构：</string>
    </property>
   </widget>
   <widget class="QLabel" name="pkgVersionlabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>软件包版本：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="developmentKernelVersionLineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="developmentOsVersionLineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="developmentIoModuleVersionLineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="developmentKernelVersionLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>内核版本：</string>
    </property>
   </widget>
   <widget class="QLabel" name="developmentFirmwareVersionLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>固件版本：</string>
    </property>
   </widget>
   <widget class="QLabel" name="pkgDependencylabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>软件包依赖库：</string>
    </property>
   </widget>
   <widget class="QLabel" name="pkgNamelabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>软件包名称：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="developmentSocVersionLineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pkgNameLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pkgDependencyLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pkgVersionLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pkgArchLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>661</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>安装运行时专用信息设备版本信息</string>
   </property>
   <widget class="QLabel" name="runtimeKernelVersionlabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>内核版本：</string>
    </property>
   </widget>
   <widget class="QLabel" name="runtimeIoModuleVersionlabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IO保密管理模块版本：</string>
    </property>
   </widget>
   <widget class="QLabel" name="runtimeOsVersionlabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>系统版本：</string>
    </property>
   </widget>
   <widget class="QLabel" name="runtimeFirmwareVersionlabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>固件版本：</string>
    </property>
   </widget>
   <widget class="QLabel" name="runtimeSocVersionlabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>安全卡版本：</string>
    </property>
   </widget>
   <widget class="QLabel" name="runtimeOsVersionLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="runtimeFirmwareVersionLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="runtimeIoModuleVersionLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="runtimeSocVersionLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="runtimeKernelVersionLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="descLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>软件基本功能描述：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="descLineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>531</y>
     <width>631</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>228</width>
        <height>28</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sureButton">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>productNameLineEdit</tabstop>
  <tabstop>descLineEdit</tabstop>
  <tabstop>developmentOsVersionLineEdit</tabstop>
  <tabstop>developmentKernelVersionLineEdit</tabstop>
  <tabstop>developmentSocVersionLineEdit</tabstop>
  <tabstop>developmentFirmwareVersionLineEdit</tabstop>
  <tabstop>developmentIoModuleVersionLineEdit</tabstop>
  <tabstop>installReasonLineEdit</tabstop>
  <tabstop>installResultLineEdit</tabstop>
  <tabstop>installWarningLineEdit</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>sureButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
